docs/sdkx_android/getting-started.mdx

```diff
@@ -32,21 +32,45 @@ Android app right away.
 
 If you are on SDK X or migrating to SDK X please note the following requirements:
 
-* SDK X is functional only above API Level 21 i.e Android OS 5 Lollipop and above. [Update (August 2023): SDK X 10.2.x is functional only above API Level 24 i.e Android OS 7 Nougat and above]
+* SDK X is functional only above API Level 24 i.e Android OS 7 Nougat and above.
 * The `minSDKVersion` in code is kept as API Level 16 to allow importing the SDK and compiling in apps that allow installation till API Level 16 i.e Android 4.4 Jelly Bean.
-* If your app is installed on a device below API level 21 (i.e below Android OS 5 Lollipop) then you can choose to have a different medium for providing support. For example, Email, Web etc. We throw an `UnsupportedOSVersionException` if the SDK is initialized below API Level 21. Please catch and use this exception according to your needs. Refer: [Initializing the SDK](/sdkx_android/getting-started/#initializing).
+* If your app is installed on a device below API level 24 (i.e below Android OS 7 Nougat) then you can choose to have a different medium for providing support. For example, Email, Web etc. We throw an `UnsupportedOSVersionException` if the SDK is initialized below API Level 24. Please catch and use this exception according to your needs. Refer: [Initializing the SDK](/sdkx_android/getting-started/#initializing).
+* Please [**refer here**](https://support.helpshift.com/hc/en/13-helpshift-technical-support/faq/1217-sdk-x-os-technical-support-policy/) for our Technical OS Support Policy
```

Comment:
Please [**refer here**](https://support.helpshift.com/hc/en/13-helpshift-technical-support/faq/1217-sdk-x-os-technical-support-policy/) for our Technical OS Support Policy.